gatorgrade/main.py

```diff
@@ -0,0 +1,18 @@
+import typer
+
+app = typer.Typer(add_completion=False)
+file = "gatorgrade.yml"
```

Comment:
Could you explain more about why `file` is a global variable here? If it's intended to be a constant, it should probably follow `pylint`'s naming guidelines for constant globals: `FILE` would be the correct name.

Reply:
I intended this to be a global variable since it is something I assumed mutliple commands would probably use, so I thought defining it once and using it from there could be a good idea. I changed the name to be all capitalized to fit better with pylint's naming guidelines!